src/Pim/Component/Catalog/Factory/ProductValue/ProductValueFactoryInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Pim\Component\Catalog\Factory\ProductValue;
+
+use Pim\Component\Catalog\Model\AttributeInterface;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+
+/**
+ * Factory that creates product values.
+ *
+ * @internal  Interface for the factories used internally by \Pim\Component\Catalog\Factory\ProductValueFactory.
```

Comment:
I think it does. Integrators will make their own factory if they need one, but when creating product values, they should never call these factories, but only the main one.